### PR TITLE
freemiumaccounts.net

### LINKED ISSUE #72727: freemiumaccounts.net
### Issue URL (Anti Adblock Script)
 [https://freemiumaccounts.net/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Ffreemiumaccounts.net%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.3
Browser: | Chrome
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Strip URLs from tracking parameters, <br/>Self-destructing third-party cookies (60), <br/>Block Push API, <br/>Block Location API
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>Peter Lowe's Blocklist, <br/>AdGuard Mobile Ads, <br/>AdGuard Base<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> Adblock Warning Removal List, <br/>I don't care about cookies, <br/>EasyList Cookie List, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist, <br/>Spam404, <br/>NoCoin Filter List<br/><br/><b>Other:</b><br/> AdGuard DNS
Custom filter subscriptions: | https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/annoyances.txt,<br>https://raw.githubusercontent.com/uBlockOrigin/uAssets/master/filters/privacy.txt

### PR DESCRIPTION
**Pull request by wout-codes**
#72727

### CODE REVIEW

File: EnglishFilter/sections/antiadblock.txt [hunk 1]
```diff
@@ -119,7 +119,7 @@ fantastic-film.online,joinfaucet.com#@#.TextAd
...
...ft4win.com,davidgalaxia.com,nullslide.com,softmafia.com,toppremiumpro.com,relmz.io,healthdelay.com,wikitechy.com,quicasting.it,xn--80abmrinidrghe0b8e.xn--p1ai,senpa.io,crypto-lovers.club,bioskopkaca21.com,larvelfaucet.com,soft4win.co,watchseries.video,daviruzsystems.com,veryfiles.com,itdplus.ru,graphicdesignresources.net,freecoursenow.com,apptoko.com,freedownloadae.com,emoji.gg,lookimg.com,topfreefaucet.com,stfly.me,skidrowfull.com,pensionioggi.it,profitlink.info,apnetv.me,umecourses.xyz,udemydownload.com,text2voice.org,paste.happy2hub.net,management.ind.in,freedeepweb.blogspot.com,4download.net,eduzabawy.com,vstplugs.com,jwearn.com,fontyukle.net,zagl.info,9ketsuki.info,utorrentgamesps2.blogspot.com,onlinemovie.one,aradramatv.co,dayoftheweek.org,osuskinner.com,yoursavegames.com,aatfal.com,goo-2o.com,canadapolicereport.ca,web-telegramm.org,xenforo.pk,xia1ge.com,xiaomifirmwareupdater.com,xossip.com,xossip.rocks,xpickle.club,xshark.me,yalujailbreak.net,yggtorrent.fr,yibtccoins.com,youmath.it,yourlifeupdated.net,yourtechguider.com,yuinime.net,zermusicz.com,zetabitco.com,zeusnews.it,zlounge.co,znaki.vxm.pl,zoey-deutch.us,zone-telechargement2.net,zoogirls.org#%#//scriptlet("prevent-bab")
```
Comment (zloyden) [inline EnglishFilter/sections/antiadblock.txt:122]: Also please add `@@||freemiumaccounts.net^$generichide` to the `content_blocker.txt` -> `! START: BlockAdBlock(no JS)` near similar rules.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Comment by wout-codes** [comment]
@zloyden Done. I had to create another Pull Request as I do not know how multiple files should be added in a Pull Request.

**Comment by zloyden** [comment]
Usually, we do it in one branch, but ok in this time. What app do you use for AdGuard repo?
Try Microsoft Visual Studio Code next time, you can make changes there easily.

**Review by zloyden — APPROVED** [review]

**Comment by wout-codes** [comment]
> Usually, we do it in one branch, but ok in this time. What app do you use for AdGuard repo?
> Try Microsoft Visual Studio Code next time, you can make changes there easily.

Yes, I have Microsoft Visual Studio. I do not really use that for AdGuard. I have a lot of coding work so I might mix up

**Comment by zloyden** [comment]
Well, there is no problem to make a multiple changes in files.
You just edit it in another section or file and make a commit like usual.

**Comment by wout-codes** [comment]
Ok, sure. :)

**Pull request merged**